content/doc/book/pipeline/syntax.adoc

```diff
@@ -278,6 +278,10 @@ completion status of the Pipeline or stage.
 status of the Pipeline's or stage's run.
 `changed`:: Only run the steps in `post` if the current Pipeline's or stage's
 run has a different completion status from its previous run.
+`fixed`:: Only run the steps in `post` if the current Pipeline is successful
+and the previous run failed or was unstable.
+`regression`:: Only run the steps in `post` if the current Pipeline's status
+is failure, unstable, or aborted and the previous run was successful.
```

Comment:
These two conditions are similar to `changed`, which mentions stages — can they be mentioned here, too?

Reply:
Yeah, right, adding.